sdk/react/src/DVCProvider.tsx

```diff
@@ -12,10 +12,10 @@ type Props = {
 let client: DVCClient | undefined
 
 export default function DVCProvider(props: Props): React.ReactElement {
-    const { envKey, user, options } = props.config
+    const { sdkKey, user, options } = props.config
```

Comment:
this would need to be a major version bump

Reply:
I can probably update it to support both the `envKey` / `sdkKey` from any public interfaces.